CMakeLists.txt

```diff
@@ -127,16 +127,16 @@ if(${VELOX_BUILD_BENCHMARKS})
 endif()
 
 if(${VELOX_BUILD_PYTHON_PACKAGE})
-  set(VELOX_BUILD_TESTING OFF)
+  set(VELOX_BUILD_TESTING ON)
   set(VELOX_ENABLE_PRESTO_FUNCTIONS ON)
   set(VELOX_ENABLE_DUCKDB ON)
   set(VELOX_ENABLE_EXPRESSION ON)
   set(VELOX_ENABLE_PARSE ON)
   set(VELOX_ENABLE_EXEC ON)
-  set(VELOX_ENABLE_AGGREGATES OFF)
-  set(VELOX_ENABLE_HIVE_CONNECTOR OFF)
-  set(VELOX_ENABLE_TPCH_CONNECTOR OFF)
-  set(VELOX_ENABLE_SPARK_FUNCTIONS OFF)
+  set(VELOX_ENABLE_AGGREGATES ON)
```

Comment:
Currently we need to enable hive and tpch too even though they are not used as otherwise the build fails (there are cross dependencies).

Reply:
Would be nice to understand where these dependencies come from. As first it seems like these should be orthogonal.